src/test/resources/net/sf/jabref/importer/fileformat/MedlineImporterTestSingleEntry.xml

```diff
@@ -1,7 +1,7 @@
-<PubmedArticleSet>
```

Comment:
Why not adding a second test case? - Modified test files always cause a strange feeling